website/docs/guides/best-practices/environment-setup/1-env-guide-overview.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "dbt Cloud environment best practices"
+id: 1-env-guide-overview
+description: Learn how to configure environments in dbt Cloud.
+displayText: "dbt Cloud environment best practices"
+hoverSnippet: Learn how to configure environments in dbt Cloud.
+---
+
+> *How do I manage environments in my dbt Cloud project? How many do I need?*
+>
+> *How does my data warehouse structure map to environments in dbt Cloud?*
+>
+> *What do git branches have to do with my dbt Cloud environments?*
+>
+
+If these questions keep you up at night, you’ve come to the right place! When it comes to managing your dbt Cloud environments, there is not a one-size-fits-all solution for all teams. In this guide we’ll walk you through a few environment architecture options for dbt Cloud that we’d recommend, and hopefully you find an option that works for you.
+
+## Learning goals
+
+This guide has three main goals:
+
+- Provide our recommendations on managing dbt Cloud environments
+- Illustrate these recommendations with comprehensive examples
+- At each stage, explain *why* we recommend the approach that we do, so that you're equipped to decide when and where to deviate from these recommendations to better fit your organization’s unique needs
+
+:::info
+☁️ This guide focuses on architecture for **dbt Cloud**. However, similar principles apply for developers using dbt Core. Before diving into this guide we recommend taking a look at our **[dbt Cloud environments](docs/collaborate/environments/dbt-cloud-environments)** page for more context.
+
+:::
+
+### How many environments do I really need?
+
+Environments define the way that dbt will execute your code, including
+
+- the **version of dbt** that will run.
+- the **version of your code** to be executed.
+- the **connection information** for your warehouse.
+- In dbt Cloud, there are **two types of environments:**
+  - **Development** - the environment settings in which you work in the IDE on a development branch.
+  - **Deployment** - the environment settings in which a dbt Cloud job runs.
+
+In this guide, we’re going to focus on **deployment environments**, which determine how your project is executed when a **dbt Cloud job executes**.
+
+Depending on your desired outcome, the number of deployment environments in your dbt Cloud Project will vary. Let’s dive in to see how many deployment environments works best for you.
```

Comment:
I think this paragraph should explain that there are basically two options folks can take: either one deployment or many, and they vary on your git strategies and technical competencies. The final sentence "Let's dive in to see how many deployment environments work best for you" I don't think makes a ton of sense with the table below.

Reply:
Maybe like "We provide a high overview of how these two deployment strategies work, but use the separate sections of this guide to get a deep-dive into how these setups differ and work."

Reply:
rewritten!